### ISSUE TITLE
[Config Support]: Can't get audio recording to work, ffmpeg errors

### ISSUE THREAD

**Issue by Flaxee** [blakeblackshear/frigate#3238]
### Describe the problem you are having

I'm trying to make Frigate record my camera with audio, but I'm having some trouble getting it to work. FFmpeg throw out some errors and exits. The default configuration (no audio) works fine.

The camera audio codec is PCM_ALAW, which isn't supported in a MP4 container. So the audio must be encoded as AAC. I also tried decreasing the bitrate by adding `-b:a 64k` to the output_args>record, but that didn't make any difference.

### Version

0.11.0-d2c3cdc

### Frigate config file

```yaml
mqtt:
  host: eclipse-mosquitto
  user: mosquitto
  password: password

database:
  path: /db/frigate.db

logger:
  default: info

detectors:
  cpu1:
    type: cpu
    num_threads: 8

live:
  height: 1080

birdseye:
  mode: continuous

ffmpeg:
  hwaccel_args: -hwaccel qsv -qsv_device /dev/dri/renderD128 -hwaccel_output_format qsv
  output_args:
    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac

objects:
  track:
    - person

record:
  enabled: True
  events:
    retain:
      default: 7
      mode: active_objects

cameras:
  lbv141-front:
    ffmpeg:
      inputs:
        - path: rtsp://user:password@192.168.20.187:554/stream1 # <----- 1920x1080
          roles:
            - record
        - path: rtsp://user:password@192.168.20.187:554/stream2 # <----- 640x360
          roles:
            - rtmp
            - detect
    rtmp:
      enabled: True
    detect:
      enabled: True
      width: 640
      height: 360
      fps: 5
    motion:
      mask:
        - 231,29,236,0,0,0,0,30
        - 390,160,364,301,312,297,294,153
```

### Relevant log output

```shell
[2022-05-20 10:18:15] detector.cpu1                  INFO    : Starting detection process: 224
[2022-05-20 10:18:15] frigate.app                    INFO    : Output process started: 226
[2022-05-20 10:18:15] frigate.app                    INFO    : Camera processor started for lbv141-front: 230
[2022-05-20 10:18:15] ws4py                          INFO    : Using epoll
[2022-05-20 10:18:15] frigate.edgetpu                WARNING : CPU detectors are not recommended and should only be used for testing or for trial purposes.
[2022-05-20 10:18:15] frigate.app                    INFO    : Capture process started for lbv141-front: 232
[2022-05-20 10:18:15] ws4py                          INFO    : Using epoll
[2022-05-20 10:18:35] ffmpeg.lbv141-front.record     ERROR   : Guessed Channel Layout for Input Stream #0.1 : mono
[2022-05-20 10:18:35] ffmpeg.lbv141-front.record     ERROR   : [aac @ 0x5611990cf980] Too many bits 8832.000000 > 6144 per frame requested, clamping to max
[2022-05-20 10:18:35] ffmpeg.lbv141-front.record     ERROR   : [mp4 @ 0x5611991f8140] Application provided invalid, non monotonically increasing dts to muxer in stream 1: 1024 >= 468
[2022-05-20 10:18:35] ffmpeg.lbv141-front.record     ERROR   : av_interleaved_write_frame(): Invalid argument
[2022-05-20 10:18:35] watchdog.lbv141-front          INFO    : Terminating the existing ffmpeg process...
[2022-05-20 10:18:35] watchdog.lbv141-front          INFO    : Waiting for ffmpeg to exit gracefully...
```

### Frigate stats

```json
{"detection_fps":0.0,"detectors":{"cpu1":{"detection_start":0.0,"inference_speed":33.4,"pid":224}},"lbv141-front":{"camera_fps":5.0,"capture_pid":232,"detection_fps":0.0,"pid":230,"process_fps":5.0,"skipped_fps":0.0},"service":{"latest_version":"0.10.1","storage":{"/dev/shm":{"free":133.3,"mount_type":"tmpfs","total":134.2,"used":0.9},"/media/frigate/clips":{"free":4270.6,"mount_type":"zfs","total":4295.0,"used":24.4},"/media/frigate/recordings":{"free":4270.6,"mount_type":"zfs","total":4295.0,"used":24.4},"/tmp/cache":{"free":996.0,"mount_type":"tmpfs","total":1000.0,"used":4.0}},"temperatures":{},"uptime":825,"version":"0.11.0-d2c3cdc"}}
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

CPU (no coral)

### Any other information that may be helpful
FFprobe output (from Proxmox host, therefore the ffprobe version doesn't match the version inside the Frigate container, which is 4.4.1-Jellyfin)
:
```
ffprobe version 4.3.4-0+deb11u1 Copyright (c) 2007-2021 the FFmpeg developers
  built with gcc 10 (Debian 10.2.1-6)
  configuration: --prefix=/usr --extra-version=0+deb11u1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libdav1d --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librabbitmq --enable-librsvg --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --enable-pocketsphinx --enable-libmfx --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-shared
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
  libpostproc    55.  7.100 / 55.  7.100
Input #0, rtsp, from 'rtsp://user:password@192.168.20.187:554/stream1':
  Metadata:
    title           : Session streamed by "TP-LINK RTSP Server"
    comment         : stream1
  Duration: N/A, start: 0.000000, bitrate: N/A
    Stream #0:0: Video: h264 (High), yuvj420p(pc, bt709, progressive), 1920x1080, 15 fps, 13.33 tbr, 90k tbn, 30 tbc
    Stream #0:1: Audio: pcm_alaw, 8000 Hz, 1 channels, s16, 64 kb/s
```

**Comment by blakeblackshear** [comment]
Does it work without the hwaccel arguments?

**Comment by blakeblackshear** [comment]
Also, set the logging for ffmpeg to info by adding global args. Maybe we will get more useful info.

```yaml
ffmpeg:
  global_args: -hide_banner -loglevel info
  hwaccel_args: -hwaccel qsv -qsv_device /dev/dri/renderD128 -hwaccel_output_format qsv
  output_args:
    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
```

**Comment by Flaxee** [comment]
Disabling hwaccel by commenting out the hwaccel_args line doesn't help. The error message remains the same.
I set the FFmpeg loglevel to info, this is the output:
```
```

**Comment by blakeblackshear** [comment]
Something seems to be broken with aac encoding. Hopefully this is the same issue several people are reporting. I will see if I can reproduce it.

**Comment by Flaxee** [comment]
Okay, perfect. Thanks a lot!
I should also have mentioned that it works fine on `image: blakeblackshear/frigate:stable-amd64`, with hwaccel both on and off.
So I guess it's something broken with the new ffmpeg version in 0.11.0 Beta 2.

**Comment by Flaxee** [comment]
I installed the latest Jellyfin-ffmpeg 5.0.1-4 ([jellyfin-ffmpeg5_5.0.1-4-bullseye_amd64.deb](https://repo.jellyfin.org/releases/server/debian/versions/jellyfin-ffmpeg/5.0.1-4/)) inside the container, and now it works perfectly fine to record with `-c:a aac`.

The `-stimeout` option for rtsp has been renamed to `-timeout` in ffmpeg 5.0, so I also had to change this in the input_args as shown below.

```yaml
ffmpeg:
  global_args: -hide_banner -loglevel info
  # hwaccel_args: -hwaccel qsv -qsv_device /dev/dri/renderD128 -hwaccel_output_format qsv
  # input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -stimeout 5000000 -use_wallclock_as_timestamps 1
  input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -timeout 5000000 -use_wallclock_as_timestamps 1
  output_args:
    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
```

```
root@bff0d3bc897b:/usr/lib/jellyfin-ffmpeg# ffmpeg -version
ffmpeg version 5.0.1-Jellyfin Copyright (c) 2000-2022 the FFmpeg developers
```

I've also tested with hwaccel enabled and that works as well.
```yaml
ffmpeg:
  global_args: -hide_banner -loglevel info
  hwaccel_args: -hwaccel qsv -qsv_device /dev/dri/renderD128 -hwaccel_output_format qsv
  # input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -stimeout 5000000 -use_wallclock_as_timestamps 1
  input_args: -avoid_negative_ts make_zero -fflags +genpts+discardcorrupt -rtsp_transport tcp -timeout 5000000 -use_wallclock_as_timestamps 1
  output_args:
    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
```

Should I try with other versions of jellyfin-ffmpeg as well?

**Comment by blakeblackshear** [comment]
Interesting. I was hesitant to update to 5 because it was just recently added.

**Comment by Flaxee** [comment]
Understood. I read through the release notes of ffmpeg 5 but didn't find much related to this. Only thing I found about AAC specifically was this:
`The default AAC encoder settings were also changed to improve quality. `

Maybe it's worth releasing a beta 3 version with ffmpeg 5 and see how that works for other users as well.

**Comment by NickM-27** [comment]
I haven't had issues previously but FWIW I built with this 5.0.1-4 and on my AMD 5700g with hw_accel and audio enabled things are working well 👍

**Comment by Bod1985** [comment]
I am experiencing this issue with 0.11.0 Beta 3, so sadly the switch to ffmpeg 4.3.2 didn't resolve for me

**Comment by hawkeye217** [comment]
There must still be some bug in ffmpeg 5 related to aac encoding. I have several cameras (2 Dahuas and 2 cheap unbranded) which all worked fine with ffmpeg 4 / frigate 0.10.x. The two cheap unbranded ones fail with jellyfin-ffmpeg 5 when trying to encode aac. If I run the ffmpeg commands with ``-an``, I don't have any issues.

My issue may have something to do with the way timestamps and segments work as well. This ffmpeg command works:

```
/opt/frigate# ffmpeg -guess_layout_max 0 -avoid_negative_ts make_zero -fflags nobuffer -flags low_delay -strict experimental  -fflags +genpts+discardcorrupt -use_wallclock_as_timestamps 1 -i http://user:password@192.168.1.10/videostream.asf -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 0  -strftime 1 -c:v copy -ac 1 -b:a 8k -c:a aac foo-%Y%m%d%H%M%S.mp4
```

But this one fails:

```
/opt/frigate# ffmpeg -guess_layout_max 0 -avoid_negative_ts make_zero -fflags nobuffer -flags low_delay -strict experimental  -fflags +genpts+discardcorrupt -use_wallclock_as_timestamps 1 -i http://user:password@192.168.1.10/videostream.asf -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1  -strftime 1 -c:v copy -ac 1 -b:a 8k -c:a aac foo-%Y%m%d%H%M%S.mp4
```

Only difference is the ``-reset_timestamps`` flag.

The failure message is pretty generic and not helpful:

```
[segment @ 0x55b227fe5bc0] Opening 'foo-20220818092048.mp4' for writing
Output #0, segment, to 'foo-%Y%m%d%H%M%S.mp4':
  Metadata:
    WMFSDKVersion   : 12.0.7601.17514
    WMFSDKNeeded    : 0.0.0.0000
    IsVBR           : 1
    encoder         : Lavf59.16.100
  Stream #0:0: Video: h264 (Main) (h264 / 0x34363268), yuvj420p(pc, bt709, progressive), 1280x720, q=2-31, 1k tbr, 16k tbn
  Stream #0:1: Audio: aac (LC), 8000 Hz, mono, fltp, 8 kb/s
    Metadata:
      encoder         : Lavc59.18.100 aac
[mp4 @ 0x55b227fea680] Application provided invalid, non monotonically increasing dts to muxer in stream 1: 19670 >= 493
av_interleaved_write_frame(): Invalid argument
frame=    6 fps=0.0 q=-1.0 Lsize=N/A time=00:00:02.73 bitrate=N/A speed=3.35x    
video:37kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
[aac @ 0x55b227fd2c40] Qavg: 44044.008
Conversion failed!
/opt/frigate#
```

I've disabled audio on that particular camera for now (with ``-an``) and I'll keep digging to see if I can find a solution with ffmpeg 5.

**Comment by NickM-27** [comment]
@hawkeye217 so without the `reset_timestamps` flag it works with ffmpeg5? Are you using rtsp-simple-server by chance? 

Seeing the same error with audio when using rtsp-simple-server https://github.com/aler9/rtsp-simple-server/issues/1061

**Comment by hawkeye217** [comment]
@NickM-27 No, I'm not using rtsp-simple-server. But that's interesting that it's the same error.

Unfortunately it seems like ``- reset_timestamps 0`` prevents frigate from correctly processing the recording segments and appending them to the existing larger recording file, at least with my camera.

**Comment by NickM-27** [comment]
> @NickM-27 No, I'm not using rtsp-simple-server. But that's interesting that it's the same error.
> 
> Unfortunately it seems like `-reset_timestamps 0` prevents frigate from correctly processing the recording segments and appending them to the existing larger recording file, at least with my camera.

I see the same behavior, I am now seeing that `-write_empty_segments 1` also allows the camera to work and am going to see if the behavior is any different. Hopefully we can figure this out, since yours is happening with a straight camera could perhaps make a bug against jellyfin-ffmpeg

**Comment by hawkeye217** [comment]
For reference, here's the full output from ffmpeg:

```
/opt/frigate# ffmpeg -guess_layout_max 0 -avoid_negative_ts make_zero -fflags nobuffer -flags low_delay -strict experimental  -fflags +genpts+discardcorrupt -use_wallclock_as_timestamps 1 -i http://user:pass@192.168.1.110/videostream.asf -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1  -strftime 1 -c:v copy -ac 1 -b:a 8k -c:a aac foo-%Y%m%d%H%M%S.mp4
ffmpeg version 5.0.1-Jellyfin Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 10 (Debian 10.2.1-6)
  configuration: --prefix=/usr/lib/jellyfin-ffmpeg --target-os=linux --extra-libs=-lfftw3f --extra-version=Jellyfin --disable-doc --disable-ffplay --disable-ptx-compression --disable-shared --disable-libxcb --disable-sdl2 --disable-xlib --enable-lto --enable-gpl --enable-version3 --enable-static --enable-gmp --enable-gnutls --enable-chromaprint --enable-libdrm --enable-libass --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libdav1d --enable-libwebp --enable-libvpx --enable-libx264 --enable-libx265 --enable-libzvbi --enable-libzimg --enable-libfdk-aac --arch=amd64 --enable-libshaderc --enable-libplacebo --enable-vulkan --enable-opencl --enable-vaapi --enable-amf --enable-libmfx --enable-ffnvcodec --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvdec --enable-nvenc
  libavutil      57. 17.100 / 57. 17.100
  libavcodec     59. 18.100 / 59. 18.100
  libavformat    59. 16.100 / 59. 16.100
  libavdevice    59.  4.100 / 59.  4.100
  libavfilter     8. 24.100 /  8. 24.100
  libswscale      6.  4.100 /  6.  4.100
  libswresample   4.  3.100 /  4.  3.100
  libpostproc    56.  3.100 / 56.  3.100
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
[h264 @ 0x5654f6d3bbc0] non-existing PPS 0 referenced
    Last message repeated 1 times
[h264 @ 0x5654f6d3bbc0] decode_slice_header error
[h264 @ 0x5654f6d3bbc0] no frame!
Input #0, asf, from 'http://user:pass@192.168.1.110/videostream.asf':
  Metadata:
    WMFSDKVersion   : 12.0.7601.17514
    WMFSDKNeeded    : 0.0.0.0000
    IsVBR           : 1
  Duration: N/A, start: 1660838257.300000, bitrate: N/A
  Stream #0:0: Video: h264 (Main) (h264 / 0x34363268), yuvj420p(pc, bt709, progressive), 1280x720, 1k tbr, 1k tbn
  Stream #0:1: Audio: adpcm_ima_wav ([17][0][0][0] / 0x0011), 8000 Hz, 1 channels, s16p, 32 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #0:1 -> #0:1 (adpcm_ima_wav (native) -> aac (native))
Press [q] to stop, [?] for help
[segment @ 0x5654f6e89a00] Opening 'foo-20220818105739.mp4' for writing
Output #0, segment, to 'foo-%Y%m%d%H%M%S.mp4':
  Metadata:
    WMFSDKVersion   : 12.0.7601.17514
    WMFSDKNeeded    : 0.0.0.0000
    IsVBR           : 1
    encoder         : Lavf59.16.100
  Stream #0:0: Video: h264 (Main) (h264 / 0x34363268), yuvj420p(pc, bt709, progressive), 1280x720, q=2-31, 1k tbr, 16k tbn
  Stream #0:1: Audio: aac (LC), 8000 Hz, mono, fltp, 8 kb/s
    Metadata:
      encoder         : Lavc59.18.100 aac
[mp4 @ 0x5654f6d74bc0] Application provided invalid, non monotonically increasing dts to muxer in stream 1: 23818 >= 329
av_interleaved_write_frame(): Invalid argument
frame=    5 fps=3.8 q=-1.0 Lsize=N/A time=00:00:03.26 bitrate=N/A speed=2.47x    
video:37kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
[aac @ 0x5654f6d69280] Qavg: 61048.848
Conversion failed!
```

**Comment by NickM-27** [comment]
@hawkeye217 If you could create an issue on jellyfin ffmpeg https://github.com/jellyfin/jellyfin-ffmpeg/issues that would be super helpful. I will be happy to support that issue. Previously since it was only seen with rtsp-simple-server we were unable to create an issue but if you're just seeing it with a camera it should get a full look.

**Comment by hawkeye217** [comment]
@NickM-27 No go on the issue over at jellyfin-ffmpeg. 

I'm guessing something's different with their build that's causing the issue. I am running ffmpeg 5.0.1 on my Mac and it processes the same camera stream without issue. jellyfin-ffmpeg is supposedly 5.0.1 as well.

**Comment by NickM-27** [comment]
@hawkeye217 did you create one and it got closed? That points to a jellyfin issue if other ffmpeg builds don't have the issue?

**Comment by hawkeye217** [comment]
@NickM-27 Yeah, I created one (https://github.com/jellyfin/jellyfin-ffmpeg/issues/181) and it got closed as an "upstream issue" and a suggestion to try another build. 😭

**Comment by NickM-27** [comment]
> @NickM-27 Yeah, I created one (https://github.com/jellyfin/jellyfin-ffmpeg/issues/181) and it got closed as an "upstream issue" and a suggestion to try another build. 😭
> 
> 
> 
> I didn't explicitly mention that other builds didn't have the issue. I probably should have.
> 
> 
> 
> EDIT: Just updated the issue to say that it seems to be jellyfin specific.

Do you have hwaccel enabled? What builds have you tried?

**Comment by hawkeye217** [comment]
> Do you have hwaccel enabled? What builds have you tried?

I disabled hwaccel to try to get to the root of the issue. I've mounted the latest BtbN master build into the container like Mark did on the discussion https://github.com/blakeblackshear/frigate/discussions/3513#discussioncomment-3310012. Works great, though I want to run with hwaccel so it's not the ideal workaround.

And as I mentioned, I tried the same ffmpeg command with all the same arguments as frigate on my Mac (which is just the homebrew download of ffmpeg 5.0.1) and that also works fine.

So clearly it's a jellyfin issue...
